.travis.yml

```diff
@@ -10,12 +10,13 @@ matrix:
 cache: cargo
 before_script:
   - |
-    (cargo install rustfmt || true) &&
-    export PATH=$HOME/.cargo/bin:$PATH
+    mkdir -p $HOME/.local/bin/ &&
```

Comment:
This may break stuff because of dynamic libraries which are indeed the root of all evil.

Reply:
Been having a headache with this tonight. What do you think of the latest version? Installs rustfmt-preview as a rustup component and runs the fmt check on nightly no matter which is the build installed versions.